lib/packwerk/reference_offense.rb

```diff
@@ -25,10 +29,11 @@ def initialize(reference:, violation_type:, location: nil)
 
     private
 
+    sig { params(reference: Reference, violation_type: ViolationType).returns(String) }
     def build_message(reference, violation_type)
       violation_message = case violation_type
       when ViolationType::Privacy
-        source_desc = reference.source_package ? "'#{reference.source_package}'" : "here"
```

Comment:
Same comment as https://github.com/Shopify/packwerk/pull/173/files#r790152320

Reply:
Wouldn't this change the logic? `here` will not be returned anymore.

Reply:
I may be misunderstanding something, but best I can tell `source_package` is never nil -- if it can't find a matching package, it always says the source package is root. So I think `here` is dead code.